pkg/graph/resolvers/trb_request.go

```diff
@@ -268,12 +268,12 @@ func ReopenTRBRequest(
 	}
 
 	// Check if request is already open so an unnecesary email won't be sent
-	if trb.Status != models.TRBSClosed {
+	if trb.State != models.TRBRequestStateClosed {
 		return nil, errors.New("Cannot re-open a TRB request that is not closed")
 	}
 
 	trbChanges := map[string]interface{}{
-		"status": models.TRBSOpen,
+		"status": models.TRBRequestStateOpen,
```

Comment:
Should this be modified to `state`? I didn't see a test case for `reopen`, and I think this might be failing on this case.

Reply:
Weird, I think this came over from merging main in. Fixing...